scripts/cmake/ConanSetup.cmake

```diff
@@ -31,7 +31,8 @@ endif()
 include(${PROJECT_SOURCE_DIR}/scripts/cmake/conan/conan.cmake)
 
 set(CONAN_REQUIRES
-    Boost/1.64.0@conan/stable
+    boost_optional/1.66.0@bincrafters/stable
+    boost_property_tree/1.66.0@bincrafters/stable
```

Comment:
`boost_propery_tree` includes all the other modules, see https://github.com/bincrafters/conan-boost_property_tree/blob/stable/1.67.0/conanfile.py#L15-L29

Of course it would be better to explicitly list all dependencies...